pages/index.html

```diff
@@ -4,9 +4,36 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Taingleap's Portfolio</title>
+  <link rel="stylesheet" href="../styles/index.css">
+  <link rel="stylesheet" href="../styles/about.css">
+  <link rel="stylesheet" href="../styles/contact.css">
+  <link rel="stylesheet" href="../styles/portfolio.css">
 </head>
-<body>
-  
+<body class="body">
+  <div class="div-setup">
+    <header>
+      <h1>Taingleap Chheang</h1>
+    </header>
+    <nav class="sections">
+      <a href="index.html" class="top-section">Home</a>
+      <a href="about.html" class="top-section">About</a>
+      <a href="portfolio.html" class="top-section">Portfolio</a>
+      <a href="contact.html" class="top-section"> Contact</a>
+    </nav>
+    <main>
+      <h1 class="hi">Hi,it’s me</h1>
+      <h2 class="my-name">Taingleap</h2>
```

Comment:
In semantic HTML, header tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)